src/Sylius/Component/Core/Resolver/EligibleDefaultShippingMethodResolver.php

```diff
@@ -73,6 +114,8 @@ public function getDefaultShippingMethod(ShipmentInterface $shipment): ShippingM
     }
 
     /**
+     * @deprecated
```

Comment:
Not needed as the method is private

Reply:
I added it to know what to remove in the next major version.